docs/extends.md

```diff
@@ -233,11 +233,9 @@ manually keep both environments in sync.
 ### Reference
 
 You can use `extends` on any service together with other configuration keys. It
-always expects a dictionary that should always contain two keys: `file` and
-`service`.
+always expects a dictionary that should always contain the key: `service` and optionally the `file` key.
 
-The `file` key specifies which file to look in. It can be an absolute path or a
-relative one&mdash;if relative, it's treated as relative to the current file.
+The `file` key specifies which file to look in. If the `file` key is not set then the default behaviour is to look in the current file that is specifying the extends. When setting the `file` key, it can be an absolute path or a relative one&mdash;if relative, it's treated as relative to the current file.
```

Comment:
I find "the current file that is specifying the extends" a bit confusing. How about:

> The `file` key specifies which file to look for the service in. If it's not set, Compose will look in the current file.

ping @moxiegirl for style opinions regarding em dashes